plasmapy/physics/tests/test_transport.py

```diff
@@ -37,7 +37,7 @@
 
 
 def count_decimal_places(digits):
-    '''Return the number of decimal places of the input digit string'''
+    r'''Return the number of decimal places of the input digit string'''
```

Comment:
If I've learned anything from rebasing and merging the other heavy pull requests we've got going now... PLEASE, PLEASE PLEASE keep PRs self-contained and don't start refactoring functions in other files (I'm totally guilty of this there!) unless you really need to change something meaningful. This kind of change can go with the next pull request that touches `test_transport.py`.

Reply:
You are right, of course.  I should have listened to the voice inside my head that was telling me this was a bad idea while I was doing it!  I'll revert it.